### DISCUSSION TITLE
Support for streaming

### DISCUSSION THREAD

**Discussion by srafay** [bloc97/Anime4K#186] [Ideas]
Would be super cool if it could work with videos when you are watching them online, maybe a browser extension? Sounds too futuristic I know.

**Comment by Tama47** [comment]
Actually, proof of concept has already been done [before](https://neurowhai.github.io/Anime4K/). https://github.com/bloc97/Anime4K/issues/7 and https://github.com/bloc97/Anime4K/pull/17 but the development was dropped.
